tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/asynchronous/AsyncClassLevelClient.java

```diff
@@ -41,28 +41,6 @@
 @Asynchronous
 public class AsyncClassLevelClient {
 
-
-    /**
-     * service 1 second in normal operation.
-     * @return the result as a Future
-     * @throws InterruptedException the interrupted exception
-     */
-    public Future<Connection> service() throws InterruptedException {
-
-        Connection conn = new Connection() {
-            {
-                Thread.sleep(1000);
-            }
-
-            @Override
-            public String getData() {
-                return "service DATA";
-            }
-        };
-
-        return CompletableFuture.completedFuture(conn);
-    }
-    
     /**
      * Service an operation until waitCondition is completed or 1 second timeout.
```

Comment:
I'm actually not sure why we use the 1 second timeout. All tests should be able to complete the `waitCondition` future at the right point in time. But that's definitely not for this PR to change.

Reply:
Well, we have ongoing something that allows using the config spec to pass a timeout as a parameter for all the TCKS the last pull request related to this:  https://github.com/eclipse/microprofile-fault-tolerance/pull/403. but looks that were not finished. i think we will good retake this.